doc/mkdocs/docs/api/macros/json_has_ranges.md

```diff
@@ -0,0 +1,23 @@
+# JSON_HAS_RANGES
+
+```teratermmacro
+#define JSON_HAS_RANGES /* value */
+```
+
+Wether the standard library has any support for ranges. Implies support for concepts.
+Possible values are `1` when supported or `0` when unsupported.
+
+## Default definition
+
+Unless defined by the user prior to inclusion of the `json.hpp` header, the macro definition is roughly equivalent to the follwing:
+```teratermmacro
+#if defined(__cpp_lib_ranges)
```

Comment:
I would not give a complete definition here. For `JSON_HAS_FILESYSTEM`/`JSON_HAS_EXPERIMENTAL_FILESYSTEM`, I wrote:

> The default value is detected based on the preprocessor macros `#!cpp __cpp_lib_filesystem`,
`#!cpp __cpp_lib_experimental_filesystem`, `#!cpp __has_include(<filesystem>)`, or
`#!cpp __has_include(<experimental/filesystem>)`.